lib/runtime/wasmer/imports_test.go

```diff
@@ -881,6 +881,38 @@ func Test_ext_crypto_ed25519_verify_version_1(t *testing.T) {
 	require.NotNil(t, read)
 }
 
+func Test_ext_crypto_ecdsa_generate_version_1(t *testing.T) {
+	t.Parallel()
+	instance := NewTestInstance(t, runtime.HOST_API_TEST_RUNTIME)
+
+	idData := []byte(keystore.AccoName)
+	keyStore, _ := instance.ctx.Keystore.GetKeystore(idData)
+	require.Zero(t, keyStore.Size())
+
+	mnemonic, err := crypto.NewBIP39Mnemonic()
+	require.NoError(t, err)
+
+	mnemonicBytes := []byte(mnemonic)
+	seedData, err := scale.Marshal(&mnemonicBytes)
+	require.NoError(t, err)
+
+	params := append(idData, seedData...)
+
+	encodedKeyPair, err := instance.Exec("rtm_ext_crypto_ecdsa_generate_version_1", params)
+	require.NoError(t, err)
+
+	var out []byte
+	err = scale.Unmarshal(encodedKeyPair, &out)
```

Comment:
@EclesioMeloJunior @qdm12 I would need a small favour, this function here gives 32 bytes slice while the input is 33 bytes slice. I tried looking at how scale codec works but couldn't find out why it wouldn't work.